rebalance/rebalance.py

```diff
@@ -109,11 +117,10 @@ def calc_optimal_amount(out_ours, out_total, in_ours, in_total, payload):
 
 @plugin.method("rebalance")
 def rebalance(plugin, outgoing_scid, incoming_scid, msatoshi: Millisatoshi=None,
-              maxfeepercent="0.5", retry_for="60", exemptfee: Millisatoshi=Millisatoshi(5000)):
+              maxfeepercent: float=0.5, retry_for: int=60, exemptfee: Millisatoshi=Millisatoshi(5000)):
```

Comment:
Notice that these type annotations will not really result in any type-coercion, so the casts below should probably stay in.

Reply:
Thx for the hint. I added step that force casts the arithmetic arguments, so we do not need to worry about casting later on.
I allow myself the liberty to merge the commit, as its just that and you ACKed.